workflow/realisations.py

```diff
@@ -268,145 +440,237 @@ def corners_to_array(corners_spec: list[dict[str, float]]) -> np.ndarray:
     )
 )
 
-
 POINT_SCHEMA = Schema(
-    {
-        Literal(
-            "type",
-            description="The type of the source geometry (Point, Plane or Fault)",
-        ): "point",
-        Literal(
-            "coordinates", description="The coordinates of the point source"
-        ): LAT_LON_DEPTH_SCHEMA,
-        Literal("strike", description="The strike bearing of the point source"): And(
-            float, is_valid_bearing
-        ),
-        Literal("dip", description="The dip angle of the point source"): And(
-            float, is_valid_bearing
+    And(
```

Comment:
Would it make sense to have these SCHEMA defenitions in a separate file just so this realisations.py doesn't blow up to be massive when extended?

Reply:
yeah maybe a schemas module?